lib/engine/step/g_18_tn/buy_company.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+require_relative '../buy_company'
+
+module Engine
+  module Step
+    module G18TN
+      class BuyCompany < BuyCompany
+        def can_buy_company?(entity)
+          companies = @game.purchasable_companies
+
+          entity == current_entity &&
+            (@game.turn == 1 || (@game.phase. available?('3') && @game.phase.buy_companies)) &&
+            companies.any? &&
+            companies.map(&:min_price).min <= entity.cash
+        end
+
+        def blocks?
+          @opts[:blocks]
+        end
+
+        def description
+          'Buy Companies'
+        end
+
+        def pass_description
```

Comment:
it looks like you just copy and pasted step/buy_company

can you clean this up?

Reply:
Now it has been cleaned up.